docs/UserGuide.md

```diff
@@ -334,7 +334,8 @@ Format: `clear`
 ### Undoing previous command : `undo`
 
 Restores the patient and appointment book to the state before the previous *undoable* command was executed.
-
+`undo` command will not work if you restart Nuudle.
+ 
```

Comment:
Would the following phrasing be better? 
`undo` command cannot reverse a command execution from the previous Nuudle session.